src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -1906,7 +1909,8 @@ private IOperation CreateBoundMultipleLocalDeclarationsBaseOperation(BoundMultip
                     variableDeclaration,
                     isAsynchronous: usingDecl.AwaitOpt is object,
                     usingDecl.DisposeMethodOpt.GetPublicSymbol(),
-                    _semanticModel,
+                    CreateDisposeArguments(usingDecl.DisposeMethodOpt, usingDecl.LocalDeclarations[0].DeclaredTypeOpt?.Type, usingDecl.Syntax, usingDecl.Binder),
```

Comment:
>DeclaredTypeOpt [](start = 102, length = 15)

This looks fragile, can this be null? Perhaps it would be more robust to pull the type from the local symbol?

Reply:
We don't have a local symbol for the using declaration. I've split it out and added an assert that shows we always have a type if we found a dispose method.

---
In reply to: [539463139](https://github.com/dotnet/roslyn/pull/49842#discussion_r539463139) [](ancestors = 539463139)

Reply:
> We don't have a local symbol for the using declaration.

What about the ```BoundLocalDeclaration.LocalSymbol``` property?

---
In reply to: [539652669](https://github.com/dotnet/roslyn/pull/49842#discussion_r539652669) [](ancestors = 539652669,539463139)